tests/accesscontrol/tests/access_control_sys_ptrace.go

```diff
@@ -0,0 +1,180 @@
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/deployment"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/execute"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/namespaces"
+)
+
+var _ = Describe("Access-control sys-ptrace-capability ", func() {
+
+	execute.BeforeAll(func() {
+		By("Define tnf config file")
+		err := globalhelper.DefineTnfConfig(
+			[]string{tsparams.TestAccessControlNameSpace},
+			[]string{tsparams.TestPodLabel},
+			[]string{},
+			[]string{})
+		Expect(err).ToNot(HaveOccurred(), "error defining tnf config file")
+
+	})
+
+	BeforeEach(func() {
+		By("Clean namespace before each test")
+		err := namespaces.Clean(tsparams.TestAccessControlNameSpace, globalhelper.APIClient)
+		Expect(err).ToNot(HaveOccurred())
+
+	})
+
+	// 54657
+	It("one deployment, one deployment, one pod, namespace sharing not enabled", func() {
+		By("Define deployment with shareProcessNamespace set to false")
+		dep, err := tshelper.DefineDeployment(1, 1, "accesscontroldeployment")
+		Expect(err).ToNot(HaveOccurred())
+
+		deployment.RedefineWithShareProcessNamespace(dep, false)
+
+		err = globalhelper.CreateAndWaitUntilDeploymentIsReady(dep, tsparams.Timeout)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Start test")
+		err = globalhelper.LaunchTests(
+			tsparams.TestCaseNameAccessControlSysPtraceCapability,
+			globalhelper.ConvertSpecNameToFileName(CurrentSpecReport().FullText()))
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Verify test case status in Junit and Claim reports")
+		err = globalhelper.ValidateIfReportsAreValid(
+			tsparams.TestCaseNameAccessControlSysPtraceCapability,
+			globalparameters.TestCasePassed)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	// 54658
+	It("one deployment, one pod with namespace sharing enabled, one container with SYS_PTRACE allowed", func() {
+		By("Define deployment with shareProcessNamespace set to true and SYS_PTRACE enabled")
+		dep, err := tshelper.DefineDeployment(1, 1, "accesscontroldeployment")
+		Expect(err).ToNot(HaveOccurred())
+
+		deployment.RedefineWithShareProcessNamespace(dep, true)
+
+		deployment.RedefineWithSysPtrace(dep)
+
+		err = globalhelper.CreateAndWaitUntilDeploymentIsReady(dep, tsparams.Timeout)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Start test")
+		err = globalhelper.LaunchTests(
+			tsparams.TestCaseNameAccessControlSysPtraceCapability,
+			globalhelper.ConvertSpecNameToFileName(CurrentSpecReport().FullText()))
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Verify test case status in Junit and Claim reports")
+		err = globalhelper.ValidateIfReportsAreValid(
+			tsparams.TestCaseNameAccessControlSysPtraceCapability,
+			globalparameters.TestCasePassed)
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	// 54659
+	It("one pod with namespace sharing enabled, one container with SYS_PTRACE not allowed [negative]", func() {
```

Comment:
Missing "one deployment" at the beginning to make the style coherent among test cases.